src/reporting/Reporting.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.4.13;
+
+
+library Reporting {
+    uint256 private constant AUTOMATED_REPORTING_DURATION_SECONDS = 3 days;
+    uint256 private constant AUTOMATED_REPORTING_DISPUTE_DURATION_SECONDS = 3 days;
+    uint256 private constant REPORTING_DURATION_SECONDS = 27 * 1 days;
+    uint256 private constant REPORTING_DISPUTE_DURATION_SECONDS = 3 days;
+    uint256 private constant CLAIM_PROCEEDS_WAIT_TIME = 3 days;
+    uint256 private constant REGISTRATION_TOKEN_BOND_AMOUNT = 1 ether;
```

Comment:
Is it not possible to expose these directly (without writing a function) by making them `internal` instead of `private`?  I would be OK with going against our normal patterns for this use-case (centrally defined constants) if it works.

Reply:
That unfortunately doesn't work :[ This method was the on;y way to make them accessible from a shared source that I could find